### PR TITLE
Add a CI check for an out-of-date yarn.lock

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
     "@babel/runtime": "^7.8.4",
     "@types/jest": "^26.0.20",
     "babel-jest": "^26.6.3",
-    "codecov": "^3.6.5",
```
Comment (silasbw) [inline package.json:54]: I did this to test the CI check -- I will remove before merging.

**Comment by silasbw** [comment]
Motivation was this PR https://github.com/valora-inc/wallet/pull/993 where I forgot to update the lock file

**Review by noisypigeon — APPROVED** [review]
LGTM! Nitpick, I'd probably change "up-to-date" to "Up-to-date" just to follow the capitalization pattern of the rest of the suite.

**Comment by MuckT** [comment]
Will this check still be shown as passing if the yarn file is out of date?

**Comment by silasbw** [comment]
@MuckT, yes see https://github.com/valora-inc/wallet/pull/1057/commits/d61bce687062d05273c7f36a77d7eeeab16b231c

@noisypigeon I'll apply the nit!

**Review by MuckT — APPROVED** [review]

**Pull request merged**